lib/common/typeorm.utils.ts

```diff
@@ -110,15 +110,17 @@ export function getEntityManagerToken(
 export function handleRetry(
   retryAttempts = 9,
   retryDelay = 3000,
+  connectionName = DEFAULT_CONNECTION_NAME,
 ): <T>(source: Observable<T>) => Observable<T> {
   return <T>(source: Observable<T>) =>
     source.pipe(
-      retryWhen(e =>
+      retryWhen((e) =>
         e.pipe(
           scan((errorCount, error: Error) => {
             logger.error(
-              `Unable to connect to the database. Retrying (${errorCount +
-                1})...`,
+              `Unable to connect to the database (${connectionName}). Retrying (${
+                errorCount + 1
+              })...`,
```

Comment:
Very minor: can we show the old message IF `connectionName === DEFAULT_CONNECTION_NAME`? I believe that there's no benefit in showing `(default)` for most users.